### PR TITLE
bpo-21063: Improve module synopsis

### PR DESCRIPTION
**Pull request by GeniusLearner**
Cleanup synopsis of modules in documentation.

https://bugs.python.org/issue21063

### CODE REVIEW

File: Doc/distutils/apiref.rst [hunk 1]
```diff
@@ -2039,7 +2039,7 @@ This is described in more detail in :pep:`301`.
 ===================================================================
 
 .. module:: distutils.command.check
-   :synopsis: Check the metadata of a package
+   :synopsis: Check the meta-data of a package
```
Comment (CuriousLearner) [inline Doc/distutils/apiref.rst:2042]: I think this line was correct in the first place. Is there a reason to change `metadata` to `meta-data`?

**Review by CuriousLearner — COMMENTED** [review]
I've one question mentioned inline. Rest everything looks good to me.

File: Doc/distutils/apiref.rst (hunk 1 above)
Comment (GeniusLearner) [reply Doc/distutils/apiref.rst:2042]: I did it for consistency, please checkout the given line.

https://github.com/python/cpython/pull/17363/files#diff-007e8820cfbca8cecb1463f7c65c19b4L2038

File: Doc/distutils/apiref.rst [hunk 2]
```diff
@@ -1699,7 +1699,7 @@ lines, and joining lines with backslashes.
 ===================================================================
 
 .. module:: distutils.cmd
-   :synopsis: This module provides the abstract base class Command. This class
+   :synopsis: Provides the abstract base class Command. This class
```
Comment (brettcannon) [inline Doc/distutils/apiref.rst]:
```suggestion
   :synopsis: Provides the abstract base class :class:`~distutils.cmd.Command`. This class
```

**Review by brettcannon — CHANGES_REQUESTED** [review]
One minor improvement to (try to) make. Otherwise LGTM!

**Comment by GeniusLearner** [comment]
I have made the requested changes; please review again

**Review by CuriousLearner — APPROVED** [review]
LGTM 🌮

**Comment by CuriousLearner** [comment]
Hey @GeniusLearner 

Since this is your first contribution, you can put your name in https://github.com/python/cpython/blob/master/Misc/ACKS file which contains list of contributors to the repo, if you prefer.

**Review by eamanu — APPROVED** [review]
LGTM.

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
@GeniusLearner thanks for the PR (and everyone else for the PR review)!

**Pull request merged**